### DISCUSSION TITLE
How to download the Rapid Adapter Builder (RAB) extension

### DISCUSSION THREAD

**Discussion by muthamil16ev** [oracle-samples/oic-rab-vscode#20] [General]
Can you please show how to download the Rapid Adapter Builder (RAB) extension, we have tried but unable to download the RAB extension.

**Comment by sebastien-ma** [comment]
It's available in VS Code Marketplace now (https://marketplace.visualstudio.com/items?itemName=Oracle.oic-rapid-adapter-builder)
